client/js/Core/main.js

```diff
@@ -1,70 +1,74 @@
-var scHelper = null;
-var scKeynodes = null;
+const scHelper = null;
+const scKeynodes = null;
 const currentYear = new Date().getFullYear();
 
-SCWeb.core.Main = {
+function ScClientCreate() {
+    let res, rej;
+    let scClient = new sc.ScClient("ws://localhost:8090/ws_json");
```

Comment:
Move to config or var

Reply:
It is not fixable now, so I made it as var